app/services/support_interface/provider_access_controls.rb

```diff
@@ -0,0 +1,76 @@
+module SupportInterface
+  class ProviderAccessControls
```

Comment:
Good place to split! This class might have a more descriptive name — `ProviderAccessControlsStats` or sth?